### PR TITLE
Changes related to GitHub actions

### PR DESCRIPTION
**Pull request by mtitov**
Keep in sync with RP

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -10,37 +10,41 @@ on:
...
-
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
     - name: Set up Python 3.6
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: 3.6.13
```
Comment (andre-merzky) [inline .github/workflows/python-app.yml:23]: Just  curious: what is the reason for that change?

**Review by andre-merzky — APPROVED** [review]
LGTM

File: .github/workflows/python-app.yml (hunk 1 above)
Comment (mtitov) [reply .github/workflows/python-app.yml:23]: no significant reason, just thought to keep the same as for RP. Should I revert back to `3.6`? (and within some active PR will do the same for RP?)

Comment (andre-merzky) [reply .github/workflows/python-app.yml:23]: I don't have a strong opinion, really - but in general think we should use what users would get by default with 3.6, which is presumable the latest subminor.  But maybe not, who knows / who cares :-P

Having the repos in sync is good, so :+1: on that.

Comment (mtitov) [reply .github/workflows/python-app.yml:23]: Same here, no strong opinion :) ok, then keep it as it is for now! Thanks!

**Pull request merged**